main.cpp

```diff
@@ -52,87 +51,96 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  guitarist
+//  action 1:  the guitarist plays guitar
+guitarist.playGuitar();
+//  action 2:  the guitarist uses guitar padels
+guitarist.useGuitarPadels();
+//  action 3:  the guitarist wirtes songs
+guitarist.wirteSongs();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  cat
+//  action 1:  the cat eats fish
+cat.eatFish();
+//  action 2:  the cat runs
+cat.run();
+//  action 3:  the cat drinks water
+cat.drinkWater();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  egnineer
+//  action 1:  the engineer writes code
+engineer.writeCodes();
+//  action 2:  the engineer designs circuits
+engineer.designCircuit();
+//  action 3:  the engineer cooperates with team
+engineer.cooperateWithTeam();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  basketballPlayer
+//  action 1:  basketballPlayer plays games
+basketballPlayer.playGame();
+//  action 2:  basketballPlayer practices
+basketballPlayer.practice();
+//  action 3:  basketballPlayer watches films
+basketballPlayer.watchFilm();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  manufacturer
+//  action 1:  manufacturer produces
+    manufacturer.produce();
+//  action 2:  manufacturer manufacures products
+    manufacturer.manufacureProduct();
+//  action 3:  manufacturer collaborates with brands
+    manufacturer.CollaborateWithBrand();
+      
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  phone
+//  action 1:  phone rings
+phone.ring();     
+//  action 2:  phone transmitts information
+phone.transmitInformation();  
+//  action 3:  phone receives texts
+phone.receiveText();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  wind
+//  action 1:  wind blows
+wind.blow();
+//  action 2:  wind forms hurricane
+wind.formHurricane();
+//  action 3:  wind brings flower seeds
+wind.bringFlowerSeed();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  speaker
+//  action 1:  speaker plays
+speaker.play();  
+//  action 2:  speaker connect with phone
+speaker.connectPhone();
+//  action 3:  speaker vibrates
+speaker.Mute();
 
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  microphone
+//  action 1:  microphone receives sounds
+microphone.receivesSound();   
+//  action 2:  microphone tests sound level
+microphone.testSoundLevel();
+//  action 3:  microphone collects signal
+microphone.collectSignal();  
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.